samples/tfm_integration/tfm_psa_test/boards/nrf9160dk_nrf9160ns.overlay

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* Disable uart1 in nonsecure since it is used by the TFM secure app. */
+&uart1 {
```

Comment:
If my PT for nrf9160 goes in first, we will not need this.

Reply:
It went in now, right?

Reply:
Yes.

Reply:
Removed